srcpkgs/mgba/template

```diff
@@ -1,18 +1,21 @@
 # Template file for 'mgba'
 pkgname=mgba
-version=0.8.4
+version=0.9.2
 revision=1
 build_style=cmake
 hostmakedepends="pkg-config qt5-host-tools qt5-qmake desktop-file-utils"
-makedepends="SDL2-devel ffmpeg-devel libedit-devel libepoxy-devel libmagick-devel
- libzip-devel qt5-multimedia-devel qt5-tools-devel"
+makedepends="SDL2-devel ffmpeg-devel libedit-devel libepoxy-devel libmagick-devel libzip-devel qt5-multimedia-devel qt5-tools-devel"
 short_desc="Game Boy Advance emulator written in C"
 maintainer="bra1nwave <bra1nwave@protonmail.com>"
 license="MPL-2.0"
 homepage="https://www.mgba.io/"
 changelog="https://raw.githubusercontent.com/mgba-emu/mgba/master/CHANGES"
 distfiles="https://github.com/mgba-emu/${pkgname}/archive/${version}.tar.gz"
-checksum=6b94873dac9040fd6fd9f13f76dc48f342e954f3b4cf82717b59601c3a32b72c
+checksum=29ca22ebc56b26a4e7224efbb5fa12c9c006563d41990afb0874d048db76add4
+
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	makedepends+=" libatomic-devel"
```

Comment:
```
/usr/lib/gcc/arm-linux-musleabihf/10.2.1/../../../../arm-linux-musleabihf/bin/ld: qt/CMakeFiles/mgba-qt.dir/library/LibraryController.cpp.o: undefined reference to symbol '__atomic_compare_exchange_8'
/usr/lib/gcc/arm-linux-musleabihf/10.2.1/../../../../arm-linux-musleabihf/bin/ld: /usr/arm-linux-musleabihf/usr/lib/libatomic.so.1: error adding symbols: DSO missing from command line
```

something goes wrong in building >.<

Reply:
looks like `/usr/arm-linux-musleabihf/usr/lib/libatomic.so.1` is missing from the compiler cli command

Reply:
Just set `LDFLAGS=-latomic` in the template, but it might not be enough. Since it's cmake, check out how uhd does it:

```
	makedepends+=" libatomic-devel"
	configure_args+=" -DCMAKE_CXX_STANDARD_LIBRARIES=-latomic"
```

Reply:
yeah, something like this looks good
let me try to implement it this way, or try other ways if it doesnt work